src/main/webapp/app/complaints/complaints-for-tutor/complaints-for-tutor.component.ts

```diff
@@ -35,6 +37,7 @@ export class ComplaintsForTutorComponent implements OnInit {
     showLockDuration = false;
     showRemoveLockButton = false;
     isLockedForLoggedInUser = false;
+    course: Course | undefined;
```

Comment:
(same in complaints-form.component.ts)
```suggestion
    course?: Course;
```

Reply:
Yes, this is a better solution.